code/datums/mood_events/drug_events.dm

```diff
@@ -7,6 +7,11 @@
 	mood_change = 2
 	timeout = 3600
 
+/datum/mood_event/wrong_brand
+	description = "<span class='warning'>I hate that brand of cigarettes.</span>\n"

```

Comment:
```suggestion
	description = "<span class='warning'>I hate that brand of cigarette.</span>\n"
```

Reply:
Previous is valid english

Reply:
🤷‍♀️ it's a suggestion, you don't consume cigarettes, you consume a cigarette, one at a time.